main/Button/stories/components/Custom.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import type {FC} from 'react';
+import type {ThemeType} from '@dooboo-ui/theme';
+import {ThemeProvider} from '@dooboo-ui/theme';
+import {Button} from '../../index';
+import {Section} from '../../../GlobalStyles';
+import {css} from '@emotion/native';
+
+const Custom: FC<{themeType: ThemeType}> = ({themeType}) => {
+  return (
+    <ThemeProvider initialThemeType={themeType}>
+      <>
+        <Section>
+          <Button
+            type="solid"
+            text="ENABLED"
+            style={{padding: 4}}
+            styles={{
+              container: css`
+                background-color: #449944;
+                padding: 12px 12px;
+              `,
+              text: {
+                color: '#fff',
+              },
```

Comment:
This would be better to be a single line when there is only single nested props.